.github/workflows/phpunit.yml

```diff
@@ -13,16 +13,14 @@ jobs:
     strategy:
       matrix:
         dependencies:
-          - "locked"
```

Comment:
Locked dependencies test should be retained - it guarantees long-term stability of the CI pipeline, and shows us when incompatibilities arise, and where